py/server/deephaven/table.py

```diff
@@ -1949,6 +1980,70 @@ def format_row_where(self, cond: str, formula: str) -> Table:
         except Exception as e:
             raise DHError(e, "failed to color format rows conditionally.") from e
 
+    def format_data_bar(self, column: str, value_column: str = None, min: float = None, max: float = None,
+                        axis: Union[AxisOption, str] = None, positive_color: Union[str, List[str]] = None,
+                        negative_color: Union[str, List[str]] = None, value_placement: Union[ValuePlacementOption, str] = None,
+                        direction: Union[DirectionOption, str] = None, opacity: float = None, marker_column: str = None,
+                        marker_color: str = None) -> Table:
```

Comment:
There is a color type that should be supported and included in the type hints.
https://deephaven.io/core/pydoc/code/deephaven.plot.color.html

Reply:
Was getting a circular import error when I tried using the Color type from deephaven.plot. I do still verify in TableDefaults.java that the str passed in is a valid color. I noticed that the color in layout hints seems to use str as well so I'm not if this is a change we still want and if so I assume I should change it for that one as well?

`Type: <class 'ImportError'>
Value: cannot import name 'Table' from partially initialized module 'deephaven.table' (most likely due to a circular import) (/Users/ethanalvizo/Deephaven/deephaven-core/py/venv/lib/python3.11/site-packages/deephaven/table.py)
`

Reply:
This may be from type hints.  See https://peps.python.org/pep-0484/#forward-references

Reply:
Still same error log when I tried 'Color' so that it would use forward references

Reply:
Then I think you are missing a change somewhere.

Reply:
Or is it circular in imports not typehints?

Reply:
Sorry my mistake on this one I needed to remove the import at the top once I changed it to use forward references. I forgot to do that before so the circular import was still technically there